maven_publish.gradle

```diff
@@ -11,6 +11,21 @@ if (publishingActive) {
                     pom {
                         name = project.pomName
                         description = project.pomDescription
+
```

Comment:
Nevyhodíme už všude použití toho maven_publish? Přijde mi, že nastavení pomName a pomDescription tam už nemá moc smysl.
`apply plugin: 'maven-publish'`

Reply:
Takže vlastně celej maven_publish.gradle a ty bloky:
```
ext {
    pomName = 'Storyous Libraries: Bills'
    pomDescription = 'Bills API'
}
apply from: '../maven_publish.gradle'
```

Reply:
Prave ze timhle se nastavuje publish pro jednotlivy moduly, takze bych to tam nechal